package.json

```diff
@@ -7,14 +7,17 @@
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "codecov:e2e": "codecov -f ./coverage/coverage-final.json",
-    "codecov:unit": "codecov -f ./coverage/jest/coverage-final.json",
-    "codecov": "codecov -f coverage/coverage-final.json && codecov -f coverage/jest/coverage-final.json",
+    "codecov:e2e": "codecov -f ./jest-coverage/coverage-final.json",
+    "codecov:unit": "codecov -f ./jest-coverage/coverage-final.json",
```

Comment:
@vickywane `codecov:unit` and `codecov` appear to be stale. Are these used anywhere?

Reply:
No. They are not being used by the application itself, except by other contributors.

I have removed the `codecov:unit` script.